api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -21,6 +21,10 @@
 	
 	public static final String DIAGNOSTIC_SERVICE_SECTIONS_VALUESET_URI = "http://hl7.org/fhir/ValueSet/diagnostic-service-sections";
 	
+	public static final String INTERPRETATION_VALUE_SET_URI = "http://hl7.org/fhir/ValueSet/observation-interpretation";
+	
+	public static final String VERY_SUSCEPTIBLE_SYSTEM_URI = "https://fhir.openmrs.org/ext/obs/interpretations";
```

Comment:
Can we construct this as we construct the other extension URLs? That way if we need to change something, we only have to change it in one place.